src/Book.js

```diff
@@ -22,6 +23,15 @@ export const Book = () => {
     setTitleState('');
   };
 
+  const createNewPage = async () => {
+    const result = await sendJson('createPage', {
+      pageDes: newPageDes
+    });
+    if (result.error) {
+      setError(result.error);
+    }
```

Comment:
I imagine you'll want to do `loadData('getPagesForBook/' + id, setPages, setError);` again if there is no error.